docs/source/compute_config/k8s_v2_job.md

```diff
@@ -0,0 +1,96 @@
+# Kubernetes v2 (batch/job)
+
+Lithops with kubernetes as serverless compute backend.
+
+This new release represents a highly optimized and efficient iteration of Kubernetes, leveraging advanced parallelization and process optimization techniques to improve task execution.
```

Comment:
Can you elavorate this? what does it mean "highly optimized and efficient iteration of Kubernetes"? and what are the "process optimization techniques"? How this differs from the k8s backend? why is this one better? Can you also add the descripion in the PR?

Reply:
The main differences between the original version and this new, more optimized version are that:
* The original version creates a pod for each of the functions, in parallel and without taking into account the number of CPUs in the Kubernetes node. Being able to launch 1000 pods at a time.
* On the other hand, the optimized version detects the number of cpus and assigns 1 pod with all the available cpus to each cluster node. Making the start more efficient, parallelization does not generate a bottleneck, and we achieve improvements in practically half the time.

As for the comment, it may not be explained as well as possible, I didn't want to clutter the documentation with technical data, but if it's better, I'll change it that way.

Reply:
IMO, I think it is not a good idea to have muiltiple backends for the same engine, since as I can see most of the code is the same. 

If this is an improved verion of the previous one, we should replace the previous one. If you want to distinct between the previous approach of spawning pods and this approach, maybe it makes much more sense to add a config parameter in the `k8s` section of the config file to activate this new feature.

Reply:
Apart of the previous comment, If I understood correctly this feature, I think you can achieve the same behaviour by setting the parameters:

```yaml
k8s:
    runtime_cpu = X  Number of CPUs in the node
    max_workers = Y   # Number of nodes
    worker_processes = X  # Number of CPUs in the node
```

These parameters are not in the config file (I put them as an example), but calculated before the invocation.
Have you tried this approach?

Reply:
I wasn't aware of these Lithops configurations, but I must say that aside from optimizations in the number of CPUs, there are also others, such as passing the function through RabbitMQ queues (making the transfer faster to each node in the cluster) and assigning identifiers to each processor core for better task distribution.

Reply:
What I meant is that: if we use the original code, and we take from you implementation only the `_get_nodes()` method, you can automatically fill `runtime_cpu `,`max_workers `and `worker_processes`, and thus maybe (I'm not sure, I didn't test it) nothing else is necessary to replicate the scenario you are posing in this PR. 

On the other hand, Lithops already integrates a multiprocessing approach based on `worker_processes`, so that if you set `worker_processes = 15`, the pod will internally get 15 tasks and start 15 processess in parallel to process those tasks. So I wonder if [the processes](https://github.com/lithops-cloud/lithops/pull/1144/files#diff-167110d4fba4e2469323b99271e4696e4400d99decefaa3b264bde6b945be36dR153-R162) you are starting in the `entry_point.py` of this PR are doing the same, so not necessary, since the lithops handler and the original entry_point.py already implements it.

Reply:
I believe that in the end, there are too many differences between the two codes and their way of functioning, such as the implementation of Rabbit, the assignment of functions, etc. But in the end, you will know which is the best way to carry out the implementation. If you'd like, you can send me an email at danielalejandro.coll@urv.cat, and we can discuss it further.